logging/upstream_log.go

```diff
@@ -149,17 +152,37 @@ func (u *UpstreamLog) RoundTrip(req *http.Request) (*http.Response, error) {
 		fields["validation"] = validationErrors
 	}
 
+	serverTimingsVal := make(utils.ServerTimings)
+	serverTimingsKey := ""
+
+	// For test cases: "backend" is sometimes not set
+	if u.log != nil {
+		if key, exists := u.log.Data["backend"]; exists {
+			serverTimingsKey = key.(string)
+		}
+	}
+
+	if name, ok := requestFields["name"].(string); ok && name != "default" {
+		serverTimingsKey += "_" + name
```

Comment:
Should we use different separators for adding
* request name
* timing name
* metric distinguisher (1, 2, ...)

? Other valid separators could be `.` or `-`.

Reply:
@malud , @filex 

What do you mean?

Reply:
The header is informational/ for debugging. I'd keep it simple and stick to _. Users will still be able to identify which metric came from which request.

(Another aspect would be accessing the metrics from JS via the ServerTiming API. Here, I like underscores, too. But I don't think that anyone will actually go that way.)